### DISCUSSION TITLE
Contributor Intros

### DISCUSSION THREAD

**Discussion by yanniboi** [LPGameDevs/EditarrrPublic#19] [General]
The purpose of this thread is to keep track of people who would like to contribute to the project. Introduce yourselves and include as much information as you are comfortable with sharing online. Not all the information needs to be relevant to this project, but we will try to find tasks that fit your expertise as well as work that will stretch you and help you grow as a developer (paired with a more experienced dev if possible).

@LPGameDevs/game-dev-field-guide

**Example**:

Name: yanniboi
Location/Timezone: UK (GMT)
Main language: English 
Other languages: German / Spanish
Usually available on discord: Weekdays 09:00-17:00 and after 20:00
Time commitment: 1 - 2 hours a week (mostly facilitating not developing)
Game Engine: Unity and Godot
Game dev experience: 2 years (lots of jams, no full release yet)

Strengths:
- Programming (general)
- CI/CD

Weaknesses:
- Art
- Shaders/Particles

Would like to learn:
- Async programming in C#/multi-threading
- Skeletal animation
- Multiplayer stuff

**Comment by alexxxchange** [comment]
Name: alex e
Location/Timezone: US EST
Main language: English

Usually available on discord: weekdays
Time commitment: 1 - 2 hours a week
Game Engine: Unity
Game dev experience: 2 years , jams and personal projects

## Strengths:
 3d, unity editor,
  audio,
 3d shaders & vfx

## Weaknesses:
pixel art, 2d

## Would like to learn:
 more c# programming
2d shaders & vfx
multiplayer

github user: https://github.com/alexxxchange

**Comment by zt1103** [comment]
Name: tuna/zt1103
Location/Timezone: US East
Main language: English
Other languages: very little of various romance languages
Usually available on discord: Whenever
Time commitment: Whatever I commit to when I commit to it, probably an hour or two a week
Game Engine: Unity (some) Godot (trying to learn)
Game dev experience: lots of dabbling, not a lot of success

Strengths:

Programming (15 years of non-game C# experience)



Weaknesses:

Art

Would like to learn:

Multiplayer stuff

**Reply by Bolrik** [reply]
Name: Fresch/Bolrik
Location/Timezone: Germany/MESZ
Main language: German
Other languages: English
Usually available on discord: (for serious conversations) Weekdays after 21:00
Time commitment: 1 - 6 hours a week
Game Engine: Unity
Game dev experience: 3 years (lots of jams)

**### Strengths**:
Programming,
3D Models (low poly and simple styles)
Static Pixel Art (No Animation!)

**### Weaknesses:**
Art
Animation
Music/Sounds

**### Would like to learn:**
Collaboration

**Comment by BastisBastis** [comment]
Name: Basris
Location/Timezone: Sweden (GMT+1)
Main language: Swedish
Other languages: English
Usually available on discord: On and off during the days, more focused 9.30-10.30 PM
Time commitment: 1-2 hour/week more some weeks less other weeks
Game Engine: Phaser (Javascript)
Game dev experience: 1 year + 10 years of on and off game dev doodling 

###Strengths:

Music
SFX
(Coding but not in C#)

###Weaknesses:

Art
Unity
Cake

###Would like to learn:

Contributing to open source repos
To wiggle my ears

**Comment by DaSeabass** [comment]
Name: Seabass
Location/Timezone: Eastern US
Main language: English
Other languages: the language of love
Usually available on discord: Weekdays somewhat from 4-6pm, possibly weekends 1-3pm
Time commitment: 1 - 2 hours a week
Game Engine: GMS2
Game dev experience: 2 years from no coding experience

Strengths:

Pixel Art
Project Management
positive vibes

Weaknesses:

Like everything else

Would like to learn:

Up for anything (game dev-wise!)

**Comment by X-0X** [comment]
Name: Alex/LX/CmonWHAT
Location/Timezone: USA/EST
Main language: English
Other languages: Spanish
Usually available on discord: This is an accurate statement :p  Best times are currently between noon and midnight EST, subject to change when I get a job
Time commitment: I can commit to an hour or two each day, at present. 
Game Engine: Unity
Game dev experience: 2 years digital, several years more of analog (board games) but nothing shipped and no paid work in the field as yet. Did successfully bring a playable board game prototype to Unpub 6 in 2016 for playtesting and feedback, co-designed and developed with a partner. 

Strengths:

Jack of all trades (some C# programming, creative writing, some technical writing, some game art, some animation, some audio design, mixing and editing, some voice work, some UI and UX design)
I think I'm good at recognizing and nurturing talent in others, and good at visualizing and communicating "big picture" project end goals; I believe I can help more specialized individuals by working as a go-between, ensuring they have what they need to succeed at their tasks and that everyone is on more or less the same page with regards to the long and short term plans. 
I can also do a bit of this or that wherever needed.
And I think I'm a decent prose writer.

Weaknesses:

Master of few. I'm not going to be the best programmer, not the best math guy, not the best artist, you get the picture. 
Also I'm actively looking for work, so my availability is likely to change when I'm successful at that. 
I'm very new to Git in particular, though yan's video and various Getting Started resources have been quite helpful. 

Would like to learn:

Git
Improve art and animation skills
Project Management 
Unreal Engine
C++
Python
Achieve fluency in Spanish
Music
All sorts of things

**Comment by HaywardMorihara** [comment]
Name: MurphysDad
Location/Timezone: US (ET)
Main language: English
Other languages: -
Usually available on Discord: Weekdays 08:00 - 09:00, 19:00 - 22:00 ET, widely varying times on weekends
Time commitment: 1 hour / week? Or maybe I'd do a few hours for a few weeks, then nothing for a period? Haha I'm really not sure and it depends on what the work is?
Game Engine: Godot
Game dev experience: 2 years (some jams, no full releases)

Strengths:

    Programming (general)
    CI/CD
    Git
    Documentation

Weaknesses:

    Unity

Would like to learn:

    More about AWS and other cloud services for personal projects
    Art
    Multiplayer stuff    
    Music
    Writing

**Comment by Supereiris** [comment]
Name: Supereiris
Location/Timezone: GMT+3
Main language: English
Other languages: Greek
Usually available on discord: Monday-Friday after 22:00 (most of the days) , weekend after 18:00
Time commitment: 1 - 2 hours a week (can happily contribute more if I have the time)
Game Engine: Unity
Game dev experience: not a lot, still learning while making games (did some game jams too)

###  Strengths

- programming including c# and java (currently studying CS in university)
- vector art, environment/background art (not only vector style). In general digital art.
- fast learner
- creative (at least I want to think so)

###  Weaknesses

- Music / SFX (complete noob)
- project management
- how to work in a team
- not the best programmer
- animation

###  Would like to learn

- how to use github
- about music and SFX (how to make them)
- more programming techniques
- animation
- pixel art
- how saving works (levels, player data, etc)

**Comment by neon-inkblast** [comment]
Name: Kiqa / Neon / Ben
Location/Timezone: Melbourne, GMT+10
Main language: English
Usually available on discord: Weekdays 09:00-17:00 and after 20:00
Time commitment: ~2 hours a week 
Game Engine: Unity and Unreal
Game dev experience: probably a year all up, over about 7 years

Strengths: 
- programming (TS/JS/C#/C++/rust/Java/Kotlin)
- CI/CD
- web stack
- cloud arch (aws/azure)
- project management

Weaknesses:
- art
- modelling
- ux

Would like to learn:
- shader graph + HLSL
- newish unity netcode package
- new ways to do shiz
- maybe write some music 🤷

**Comment by TomieKill88** [comment]
Name: TomieKill88
Location/Timezone: UKGemrany (GMT+2)
Main language: Spanish
Other languages: English / German
Usually available on discord: Weekdays 18:00 - 22:00 for calls or meetings. For quick messaging: flexible.
Time commitment: 2 - 3 / week (actually in a lot of personal projects at the moment so it can vary each week)
Game Engine: Unreal
Game dev experience: a few jams here and there, but nothing too serious

Strengths:

- Programming (C/C++, Python)
- Art (2D)

Weaknesses:

- Sound design.
- 3D Art.

Would like to learn:

- Better time management (lol)
- Project Management
- 2D animation
- Pixel Art
- Level Design
- Storytelling

**Comment by kotasnowgaming** [comment]
Name: kotasnow
Location/Timezone: EST
Main language: English
Other languages: Nonmodern Icelandic :P / Some Spanish
Sometimes available on discord: Weekdays 1700-2200 and weekends 0900-2200 periodically
Time commitment: 2-4 hours a week
Game Engine: Unity/Pygame/GMS/Unreal
Game dev experience: 1 Year of hobbying, No releases

Strengths:

All sorts of Programming (good with aprox 10 Languages) Strongest laguage being python.
Research
The different AI Agents
Learning everything but art XD
Focus / Time management

Weaknesses:

All aspects of Art :/

Would like to learn:

Blender 3D Modeling
VR Game Dev (Current personal project)
Making Unity Editor Tools
Github
Search Algorithms (basic AI's)
Working on a Game Dev project.

**Comment by DragonKJM** [comment]
_Name:_ DragonKJM
_Location/Timezone:_ UK (GMT)
_Main language:_ English
_Other languages:_ Broken Spanish
_Usually available on discord:_ Usually 18:00 - 23:00
_Time commitment:_ None for now, 3+ hours a week once coursework is done - around May/June hopefully
_Game Engine:_ Unity
_Game dev experience:_ 5 months, mostly stuff with APIs rather than in-engine

**Strengths:**

- Programming (general - C++, C#, super basic HTML/CSS)
- Teaching stuff where I can - being 5 months into learning programming, this isn't a lot

**Weaknesses:**

- Art
- Music
- Animation

**Would like to learn:**

- Anything C++/C#
- Working on a collaborative project

**Comment by CalmPewter** [comment]
Name: Calm Pewter / Julian
Location/Timezone: Germany (GMT+1)
Main language: German, English
Other languages: Italian, French
Usually available on discord: Around the clock on mobile for quick replies, usually after 22:00 for bigger stuff
Time commitment: 4 - 8 hours a week
Game Engine: Unity
Game dev experience: 5 years (no full release yet)

**Strengths:**
- Programming (mainly oop with C#) and Unity Editor
- Pixel Art / Animation (mainly characters and UI)
- Game Design
- Juice

**Weaknesses:**
- Scope/time management
- 3D modeling

**Would like to learn:**
- Dynamic music
- In-depth Unity Timeline and Cinemachine stuff

**Comment by EndocrineMSc** [comment]
Name: Endocrine (Sebastian)
Location/Timezone: Germany (GMT)
Main language: German
Other languages: English
Usually available on discord: Most of the time
Time commitment: I'm not sure yet, but at least 1-2 hours a week
Game Engine: Unity
Game dev experience: 1 year (lots of jams ~1 per month - 14 right now, no full release yet)

Strengths:
Knowing my weaknesses

Weaknesses:
Not knowing my strengths


Now for real:

Strengths:

Unity programming
Mid pixel art

Weaknesses:

Shaders,
3D,
Database stuff,
Coding outside of Unity (will hopefully change soon)
Multiplayer

Would like to learn:

Good architecture,
Programming concepts and workflows,
Shadergraph
